docs/how-to/how-to-hotplug-memory-arm64.md

```diff
@@ -15,6 +15,19 @@ $ sudo .ci/aarch64/install_rom_aarch64.sh
 $ popd
 ```
 
+## Config KATA QEMU
+
+After executing the above script, two files will be generated under the directory /usr/share/kata-containers/ by default, namely kata-flash0.img and kata-flash1.img. Next we need to change the configuration file of kata qemu, which is in /opt/kata/share/defaults/kata-containers/configuration-qemu.toml by default, specify in the configuration file to use the UEFI ROM installed above. Please refer to the following configuration:
+
+```
+[hypervisor.qemu]
+
+# -pflash can add image file to VM. The arguments of it should be in format
+# of ["/path/to/flash0.img", "/path/to/flash1.img"]
+# pflashes = []
```

Comment:
Please remove this line.

Reply:
Do you want me to remove all these comments?

Reply:
No, only line `# pflashes = []`, this is the original line in the configuration file, you have changed it to `pflashes = ["/usr/share/kata-containers/kata-flash0.img", "/usr/share/kata-containers/kata-flash1.img"]`

Reply:
ok!